pinot-tools/src/main/java/com/linkedin/pinot/tools/admin/command/CreateSegmentCommand.java

```diff
@@ -272,6 +280,19 @@ public boolean execute()
       }
     }
 
+    String configDateTimeColumnName = segmentGeneratorConfig.getTimeColumnName();
+    if (_timeColumnName == null) {
```

Comment:
should we make it mandatory? Until now we haven't mandated having time column in the data.

Reply:
Fixed. Time column is not a mandatory input anymore. Added some TODOs to check that if this is not provided, use DATETIME field, and if multiple of them are found, throw an exception.